benchmarks/bench_torch_eager.c

```
@@ -0,0 +1,111 @@
+#include "torch/torch_c.h"
+#include <stdio.h>
+#include <time.h>
+
+static double now_ms(void) {
+    struct timespec ts;
+    clock_gettime(CLOCK_MONOTONIC, &ts);
+    return ts.tv_sec * 1e3 + ts.tv_nsec / 1e6;
+}
+
+static Tensor* randn_f32(int* shape, int ndim) {
+    TorchTensorOptions o = torch_options();
```

Clang (14.0.6)

[warning] variable name 'o' is too short, expected at least 3 characters

(readability-identifier-length)